rasa_sdk/interfaces.py

```diff
@@ -230,14 +231,17 @@ def name(self) -> Text:
         raise NotImplementedError("An action must implement a name")
 
     async def run(
-        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any]
+        self,
+        dispatcher: CollectingDispatcher,
```

Comment:
@erohmensing, we have a problem with the definition of this class with parameter typing, I only realized now by reviewing the code and running the tests... It is causing a circular dependency between the `interfaces.py` and` executor.py` files. Do you think it pays to just change the docs for that method and not add the typing?

Reply:
Yes, seems so. I can fix the tests by adding an `if typing.TYPE_CHECKING` check for the import, but then pytype ultimately fails. Let's still update the typo though.